marshal/admin_test.go

```diff
@@ -59,6 +60,11 @@ func (s *AdminSuite) TestRewindConsumer(c *C) {
 	rewindOffsets["test1"] = make(map[int]int64)
 	rewindOffsets["test1"][0] = 0
 
+	// We include a second partition to check the case where multiple heartbeat goroutines are
+	// instantiated; for example, we want to ensure that there is no panic on shutdown due to
+	// "close on closed channel".
+	rewindOffsets["test1"][1] = 0
```

Comment:
The `test1` topic only has a single partition. If you want a second partition you'll need to switch to `test2` which has two partitions. And at that point I'd suggest actually making the test produce some messages to it and then rewind to a different point than 0 just to ensure we don't have a bug of the class 'everybody rewinds to 0 regardless of what's in the checkpoint'.